components/ButtonContainer/ButtonContainer.js

```diff
@@ -12,10 +12,17 @@ const StyledButton = styled.button`
   height: 25px;
 `;
 
-export default function ButtonContainer({ onSpin }) {
+export default function ButtonContainer({ onSpin, counting }) {
   return (
     <StyledButtonContainer>
-      <StyledButton onClick={onSpin}>Start</StyledButton>
+      <StyledButton
+        onClick={() => {
+          onSpin();
+          counting();
```

Comment:
use handleFunction for example: 

const handleSpin = () => {  
   onSpin(); 
   counting();
}

<StyledButton onClick={handleSpin}